package.json

```diff
@@ -4,7 +4,9 @@
   "description": "Safe(r) monkeypatching for JavaScript.",
   "main": "index.js",
   "scripts": {
-    "test": "standard && tap test/*.tap.js --coverage"
+    "test": "npm run test:cjs && npm run test:mjs",
```

Comment:
I can see this being condensed into a single line for testing instead of separating out the esm named export test, open to suggestion.